index.js

```diff
@@ -388,6 +388,27 @@ const publicApi = {
     },
 
     /**
+     * If enabled, a Preact preset will be applied to
+     * the generated Webpack configuration.
+     *
+     *     Encore.enablePreactPreset()
+     *
+     * If you wish to also use preact-compat (https://github.com/developit/preact-compat)
+     * call that method with its first parameter set
+     * to true:
+     *
+     *     Encore.enablePreactPreset(true)
+     *
+     * @param {boolean} usePreactCompat
+     * @returns {exports}
+     */
+    enablePreactPreset(usePreactCompat = false) {
```

Comment:
This is the only part I'm not sure about. When I saw your code examples, I assumed the true/false was turning the entire feature on/off (like some of our other `enable()` methods). It's more verbose, but I'd prefer `enablePreactPreset({preactCompat: true})` style (with our normal validating of any options passed).

Reply:
I definitely agree with you :)